ew/static/mutations.py

```diff
@@ -537,6 +547,104 @@
+        id_mutation=ewcfg.mutation_id_deathfromabove,
+        str_name="Death From Above",
+        alias=['death', 'dfa'],
+        str_describe_self="Lightning crashes around you due to **Death From Above**.",
+        str_describe_other="Lightning crashes around them due to **Death From Above**.",
+        str_acquire="Someone next to you gets struck by lightning! Phoebus Christ! You feel a faint amount of electricity spread towards you, invading your personal space. You've gotta expel it immediately. You have developed the mutation **Death From Above**. During lightning, 10% of your damage is additionally dealt to bystanders when attacking with any non-grenade weapon.",
+        tier=1,
+        incompatible = {
+            ewcfg.mutation_id_airlock : '"Nope, you already have that mutation. It\'s even got a bunch extra tacked on."'
+        },
+        str_transplant="Dr. Dusttrap gives you a metal fork and points towards an outlet. You go to it but, like, there's a goddamn protector on this thing! You dig into the plastic covering with your nails and, somehow, chip a nail. You succeed in tearing the protector off though. In the fork goes. You have developed the mutation **Death From Above**. During lightning, 10% of your damage is additionally dealt to bystanders when attacking with any non-grenade weapon.",
+    ),
+    EwMutationFlavor(
+        id_mutation=ewcfg.mutation_id_ichumfast,
+        str_name="I Chum Fast",
+        alias=['chum', 'icf', 'chumfast', 'ichum'],
+        str_describe_self="You've got fish to catch due to **I Chum Fast**.",
+        str_describe_other="They've got fish to catch due to **I Chum Fast**.",
+        str_acquire="Aw jeez, you just can't hold it in any longer. It's too fuckin' much mannnn!!! You explode all over, unable to contain your love for fish. You've got fish to catch. You have developed the mutation **I Chum Fast**. While fishing, you will be @'d upon any !reel notification.",
+        tier=7,
+        str_transplant="Dr. Dusstrap hands you a bottle of fish oil. \"Chug.\" You comply, and god, this stuff tastes terrible. Nevertheless, you really feel yourself more connected to fish than ever before. You have developed the mutation **I Chum Fast**. While fishing, you will be @'d upon any !reel notification.",
+    ),
+    EwMutationFlavor(
+        id_mutation=ewcfg.mutation_id_scopicretinas,
+        str_name="'Scopic Retinas",
+        alias=['sr', 'telescopicretinas', 'scopic', 'orbe'],
+        str_describe_self="You can see near and far due to **'Scopic Retinas**.",
+        str_describe_other="They can see near and far due to **'Scopic Retinas**.",
+        str_acquire="Zamn, there's a fine specimen hanging out across the street! You go to hit on them, but as you walk you don't seem to gain any distance. You look down and-JEEZ, you're looking at the individual holes in the pavement. You have developed the mutation **'Scopic Retinas**. You can now !scout up to two districts away.",
```

Comment:
i don't think your flavor writing is bad by any means. but zamn is a zug trademark we can't have endless war go around saying that